### PR TITLE
Migrated the Confluence documents to rst files.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: doc/commands.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.. py:currentmodule:: lsst.ts.Dome
+
+.. _lsst.ts.Dome-commands:
+
+####################
+Lower Level Commands
+####################
+
+This document describes the commands that will be sent from the Upper Level DCS to the Lower Level
+Components. This is not a finalized list and if the need arises for it, it will be modified accordingly.
```
Comment (couger01) [inline doc/commands.rst:10]: One sentence per line please.

File: doc/commands.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| statusApSCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusLCS       |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusLWSCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusMonCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusThCS      |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
```
Comment (couger01) [inline doc/commands.rst:76]: If these tables are prone to change, maybe its better to make a CSV file and use the [csv-table directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#csv-table).

File: doc/protocols.rst [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        'driveTemperature': [20.0, 20.0, 20.0, 20.0, 20.0],
+        'encoderHeadRaw': [0.0, 0.0, 0.0, 0.0, 0.0],
+        'encoderHeadCalibrated': [0.0, 0.0, 0.0, 0.0, 0.0],
+        'resolverRaw': [0.0, 0.0, 0.0, 0.0, 0.0],
+        'resolverCalibrated': [0.0, 0.0, 0.0, 0.0, 0.0]
+        'timestamp': 0,
+      }
+    }
+
+----------------------
```
Comment (couger01) [inline doc/protocols.rst:113]: Unnecessary top header here.

File: doc/commands.rst (hunk 1 above)
Comment (wvreeven) [reply doc/commands.rst:10]: Done

File: doc/protocols.rst (hunk 3 above)
Comment (wvreeven) [reply doc/protocols.rst:113]: Done

File: doc/commands.rst (hunk 2 above)
Comment (wvreeven) [reply doc/commands.rst:76]: In the end I chose the [list-table directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table) since I needed to be able to span certain cells over two rows and I couldn't figure out how to do that with CSV.

**Comment by wvreeven** [comment]
Changes pushed. Could you have another look please?

**Comment by wvreeven** [comment]
Sorry, I just pushed one more change since I noticed that some command parameter names and units were outdated.

File: doc/index.rst [hunk 4]
```diff
@@ -10,18 +10,25 @@ Controller for the Simonyi Survey Telescope dome at Vera C. Rubin Observatory.
 
 .. _lsst.ts.Dome-using:
 
-Using lsst.ts.Dome
+Using lsst.ts.salobj
```
Comment (couger01) [inline doc/index.rst:13]: Why did you change this to salobj?

Comment (wvreeven) [reply doc/index.rst:13]: Ouch, copy and paste error. Nice catch and sorry about that!

**Comment by wvreeven** [comment]
Copy and paste error corrected.
